### DISCUSSION TITLE
Welcome to Loop Habit Tracker Discussions

### DISCUSSION THREAD

**Discussion by iSoron** [iSoron/uhabits#683] [Announcements]
Hi everyone,

Welcome to our new discussion forum for Loop Habit Tracker!

In the past, we have used GitHub Issues for everything, from bug reports to feature requests. To better organize the project, we now have a dedicated forum for questions and discussions. As always, your contribution is very welcome:

* In [**Feature Requests**](https://github.com/iSoron/uhabits/discussions/categories/feature-requests), feel free to suggest ideas for new features or small enhancements to existing features. You can also vote on existing feature requests (by giving them a thumbs-up) and leave your own comments. Your feedback is important because it will let the developers know that there is high demand for a certain feature.
* In [**Help & FAQ**](https://github.com/iSoron/uhabits/discussions/categories/help-faq),  feel free to ask questions about the app and get help from the community. If you know how to answer someone else's question, please go ahead; your contribution is highly appreciated. Just remember to follow the [GitHub Community Guidelines](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-community-guidelines).
* In [**General Discussions**](https://github.com/iSoron/uhabits/discussions/categories/general-discussion), feel free to start any discussions about the project that don't fit anywhere else.

Moving forward, GitHub Issues will only be used for bugs in the app (when something is clearly broken) or for open development tasks already in our roadmap with clear requirements. Updates about the project will be posted in [**Announcements**](https://github.com/iSoron/uhabits/discussions/categories/announcements).

**Comment by woj-tek** [comment]
Can #684 be pinned? :-)

**Reply by iSoron** [reply]
@woj-tek Pinned.

**Comment by Afshinbidar** [comment]
Hey hey guys
This is Afshin, a former happy user of your great app.
And I wanted to tell you that i am considering to uninstall your app, after the latest update.
Let me explain: i had 2 counter widget on my phone in order to add numbers in some of my habits and before this update it was done inside of my current page not in the habit app.
But now, when i want to add an integer the widget is taking me to the app and i have get out of there after i put my number!
You totally add an unnecessary step in the widget process and that is not good for ux at all!
Btw, the ui of the widget gotten way better.

Sincerely,
Afshin

**Comment by snnkrsln** [comment]
Hello dear team
I have been using the Loop Habit Tracker app for almost a year. It keeps me motivated to continue my daily targets. I am now using an iPhone, and unfortunately, your app is not available in the Apple Store. I kindly ask you to consider your current and future clients who use Iphones. I am even ready to pay if it's not free. I also believe that some more people are willing to use the app in their Iphones. Thank you so much

**Comment by jpberriz** [comment]
Hi folks.
I'm also a big fan of the app and have been using it for several years now. 
Are any plans to make it sync with a browser view?
Just wondering. 
Thanks !

**Comment by ioctl-user** [comment]
Hello!

Thanks for this great and very useful app!

For now only share-based database export is available, and in my firmware it's impossible to directly use this for backup. So, I have to export to a messenger as a file, send it, and then save it back for backup.

So, my feature request is direct database export to the file.

**Comment by tu2-atmanand** [comment]
Thank you so much for this amazing app.

I have tried several Habit tracker apps now even the paid versions. But this app is something different. A habit tracker according to me should have two essential functionalities : 
- The easiest way to record your progress.
- The best dashboard possible with a very convenient UI/UX.

And this app surprising has been the closest to these criteria and on top of that it's open-source.

All the apps I have used in the past had numerous amounts of features and high class UI. But I have realised that it's not about the features which attracts and motivates you to keep on track with your habits. But it's the simplicity of the application which makes you always stay on the app. And this specific app, some of the screens like the heatmap and the tickmarks of all the habits on the same screen makes us realise how productive consistent we were in the past few days, months or years. And this self satisfaction is the greatest factor to not loose the momentum.

So I would request the development team to please not compromise this simplicity in the future with the new features being integrated.

Thank you one again for keeping it free and open-source.
